src/runtime/device_buffer_utils.h

```diff
@@ -130,7 +130,7 @@ WEAK device_copy make_buffer_copy(const halide_buffer_t *src, bool src_host,
             c.src_stride_bytes[j] = c.src_stride_bytes[j - 1];
         }
         c.extent[insert] = dst->dim[i].extent;
-        debug(NULL) << "c.extent[" << insert << "] = " << (int)(c.extent[insert]) << "\n";
+        // debug(NULL) << "c.extent[" << insert << "] = " << (int)(c.extent[insert]) << "\n";
```

Comment:
intentional?

Reply:
Yes. This sprays all sort of useless info thru debug builds. I suspect it was added unintentionally at some point in the past.